### ISSUE TITLE
[r-package] error in demo --lgb_multiclass

### ISSUE THREAD

**Issue by zheguzai100** [Microsoft/LightGBM#713]
`model <- lgb.train(list(),dtrain,100,valids,min_data=1,learning_rate=1,objective='multiclass',metric='multi_err',num_class=3)`

`Error in lgb.call("LGBM_DatasetCreateFromMat_R", ret = handle, private$raw_data,  : 
  api error: cannot open data file 5.1`

**Comment by guolinke** [comment]
try this: https://github.com/Microsoft/LightGBM/pull/712

it seems is a R random error .

**Comment by Laurae2** [comment]
@zheguzai100 Multiclass demo runs fine here.

**Comment by Laurae2** [comment]
@guolinke I think you can close it unless the user can provide a reproducible example. The demo works fine.
